pilco/environments/continuous_mountaincar.py

```diff
@@ -0,0 +1,175 @@
+"""
+Our modification of the OpenAI Gym Continuous Mountain Car by Olivier Sigaud:
+https://github.com/openai/gym/blob/master/gym/envs/classic_control/continuous_mountain_car.py
+
+which was (ultimately) based on Sutton's implementation:
+http://incompleteideas.net/sutton/MountainCar/MountainCar1.cp
+"""
+
+import math
+
+import numpy as np
+
+import gym
+from gym import spaces
+from gym.utils import seeding
+
+
+class Mountaincar(gym.Env):
```

Comment:
Should be `MountainCar` (camel case)

Reply:
In that case I'll also change `Cartpole` to `CartPole`.